st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -514,6 +524,51 @@ def _get_pkey_object(self, key_material):
 
         raise paramiko.ssh_exception.SSHException(msg)
 
+    def _bastion_connect(self):
```

Comment:
Once you get this to work, please refactor as there is duplication in code with connect() method.